tests/integration/models/device.spec.js

```diff
@@ -2725,6 +2728,25 @@ describe('Device Model', function () {
 						.expect(promise)
 						.to.be.rejectedWith(`Release not found: ${badRelease}`);
 				});
+
+				describe('given a device that does not support multicontainer', function () {
```

Comment:
Maybe we should wrap the old tests in a `given a device that does support multicontainer`, since both the `givenADevice(beforeEach` of that `describe()` & the `givenADevice(before` in here run for this test case.
What do you think @xginn8 ?

Reply:
done, good idea.